src/collections/members/ashparsh-pandey/index.mdx

```diff
@@ -0,0 +1,14 @@
+---
+name: Ashparsh Pandey
+position: Contributor
+image_path: ./ashparsh-pandey.webp
+github: Ashparshp
+twitter: ashparsh_
+linkedin: ashparsh
+location: Uttar Pradesh, India 
+bio: Hello! :wave: CS student here, passionate about coding and creating. Exploring the tech world. Let's connect and code something awesome.
+badges: 
+    - community
```

Comment:
You are using the wrong badge
checkout his member profile - https://meshery.layer5.io/user/756c3044-ee4f-4ad6-be15-27a7967724e7?tab=badges

Reply:
ok will make the changes needed